### PR TITLE
Scully + Netlify CMS Guide

### PR DESCRIPTION
**Pull request by pjlamb12**
docs: add Netlify CMS guide

Add a guide to show users how to use Netlify CMS with their Scully app/site

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/scullyio/scully/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [x] Other... Please describe:

## What is the new behavior?
Brief guide on setting up Netlify CMS with your Scully site.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by pjlamb12** [comment]
@SanderElias @aaronfrost this is another guide for the ngConf blog that will be released soon. Thought it would be good to be in the official docs as well.

File: docs/learn/use-netlify-cms/use-netlify-cms.md [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+```bash
+$ ng new scully-netlify-cms --routing
+...
+$ ng add @scullyio/init
+...
+$ ng g @scullyio/init:blog
+...
+```
+
+I'm not going to go over each of these steps in detail; that's best left for another post. Just know that those three commands create a new Angular application, initialize Scully, and add blog support to Scully. Next up, we need to create a couple modules and components to use for routing. You can do that like this:
```
Comment (SanderElias) [inline docs/learn/use-netlify-cms/use-netlify-cms.md:24]: This is a docs page, not a post.

File: docs/learn/use-netlify-cms/use-netlify-cms.md [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+
+![Netlify CMS Landing Page](/assets/images/blog-posts/scully-netlify-cms/netlify-cms-landing-page.png)
+
+From here, you will be able to add and edit new blog posts in a user friendly rich text or markdown editor. You won't have to edit the markdown blog post files directly any more.
+
+## [Conclusion](#conclusion)
+
+Netlify CMS is a great tool for placing a lightweight CMS on top of your Scully app. All the code can live in one codebase, and everything is just backed by markdown files directly. It doesn't take much to add it to your site, and is cost effective on top of all of that.
+
+You can see an example of all this in [this GitHub repo](https://github.com/pjlamb12/scully-netlify-cms).
```
Comment (SanderElias) [inline docs/learn/use-netlify-cms/use-netlify-cms.md:202]: I'm not sure if we should link to an external resource like this?
@aaronfrost your thoughts?

File: docs/learn/use-netlify-cms/use-netlify-cms.md (hunk 1 above)
Comment (pjlamb12) [reply docs/learn/use-netlify-cms/use-netlify-cms.md:24]: Yes, I missed this in my review. I'll change that.

File: docs/learn/use-netlify-cms/use-netlify-cms.md (hunk 2 above)
Comment (pjlamb12) [reply docs/learn/use-netlify-cms/use-netlify-cms.md:202]: Just wanted to follow up on this and see what you guys think?

Comment (SanderElias) [reply docs/learn/use-netlify-cms/use-netlify-cms.md:202]: As @aaronfrost did react on this yet, I would say take it out for now. If he is ok with it we can put it back in.

Comment (pjlamb12) [reply docs/learn/use-netlify-cms/use-netlify-cms.md]: I got that external link removed

**Review by SanderElias — APPROVED** [review]
LGTM

**Pull request merged**